src/components/Tutorial/CodePreview.vue

```diff
@@ -158,14 +158,18 @@ export default {
       return this.hasRuntimePreview && this.isRuntimePreviewVisible;
     },
     runtimePreviewTitle() {
-      return this.hasRuntimePreview ? null : 'No preview available for this step.';
+      return this.hasRuntimePreview ? null : this.$t('tutorials.preview.no-preview-available-step');
     },
     togglePreviewText() {
-      return this.hasRuntimePreview ? 'Preview' : 'No Preview';
+      return this.$tc('tutorials.preview.title', this.hasRuntimePreview ? 1 : 0);
```

Comment:
![image](https://user-images.githubusercontent.com/87735557/218893831-f552d4e8-1d83-433a-b797-2bd1c6992bf3.png)
For some reason the Chinese is displaying "no preview" even when there is.

Reply:
Thanks for catching it up! this will be fixed with the new translation strings :)